AuthenticationExample/AuthenticationExample/SignInView.swift

```diff
@@ -107,7 +107,9 @@ struct SignInView: View {
             signIn()
         } label: {
             Text(signInButtonText)
+#if !os(visionOS)
                 .frame(maxWidth: .infinity)
+#endif
```

Comment:
It seemed odd having this button's width be so large in a 2D window since a window itself can be quite large.